docs_source_files/content/webdriver/driver_requirements.de.md

```diff
@@ -384,19 +373,19 @@ System.setProperty("webdriver.edge.driver", "C:/path/to/MicrosoftWebDriver.exe")
 
 ### Internet Explorer
 
-Internet Explorer was Microsoft's default browser until Windows 10, although it 
-is still included in Windows 10. Internet Explorer Driver is the only driver 
-The Selenium project aims to support the same releases
+Der Internet Explorer war Microsoft's Standardbrowser bis Windows 10,
+trotzdem ist er ebenfalls in Windows 10 inkludiert. Der Internet Explorer Driver
+ist der einzige Driver der vom Selenium Projekt mit Releases versorgt wird 
 [Microsoft considers current](//support.microsoft.com/en-gb/help/17454/lifecycle-support-policy-faq-internet-explorer).
-Older releases may work, but will be unsupported. 
+Ältere Releases können weiterhin funktionieren, werden jedoch offiziell nicht
+weiter unterstützt.
 
-While the Selenium project provides binaries for both the 32-bit and 64-bit 
-versions of Internet Explorer, there are some 
-[limitations](//jimevansmusic.blogspot.co.uk/2014/09/screenshots-sendkeys-and-sixty-four.html)
-with Internet Explorer 10 & 11 with the 64-bit driver, but using the 32-bit 
-driver continues to work well. It should be noted that as Internet Explorer
-preferences are saved against the logged in user's account, some 
-[additional setup is required](//github.com/SeleniumHQ/selenium/wiki/InternetExplorerDriver#required-configuration).
+Das Seleniumprojekt stellt die Driver für 32-bit und 64-bit Versionen des Internet
```

Comment:
Selenium projekt

Reply:
"Selenium-Projekt" would be the best correct ssolution.
I will update my pull request shortly